terraform/README.md

```diff
@@ -1,8 +1,101 @@
+This repository contains one folder for each AWS account whose infrastructure is managed through terraform. Currently,
+it supports:
+* [`development`](/development) - The Simple AWS Dev account, managing Sandbox, QA, and Security environments
+* [`bangladesh`](/bangladesh) - The Simple Bangladesh account, managing Bangladesh Demo and Bangaldesh Production
+  environments
+
+If you want to set up a new AWS account, go to [Setting Up A New AWS Account](#setting-up-a-new-aws-account). Otherwise
+go to [Getting Started](#getting-started).
+
+## Getting Started
+
+### 1. Add the master AWS SSH key to your machine
+
+* Create a blank SSH keypair in your SSH directory
+
+```bash
+$ touch ~/.ssh/simple_aws_key ~/.ssh/simple_aws_key.pub
+```
+
+* Find the "AWS Master SSH Key" in 1Password
+
+* Add the contents of the "Private key" to `simple_aws_key`
+
+* Add the contents of the "Public key" to `simple_aws_key.pub`
+
+### 2. Decrypt all encrypted terraform files
+
+Sensitive terraform configuration is stored and checked into Github as encrypted files using
+[ansible-vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html). These files are decrypted locally and
+git-ignored for local development. Before doing any development, run `./decrypt path-to-your-vault-file` to make sure to
+decrypt the latest versions of the encrypted ansible vault files in this directory. For example:
+
+```bash
+$ cd terraform
+$ ./decrypt ~/code/simple/ansible_password_file
+
+Decrypting bangladesh/bd.simple.org.chain.pem.vault to bangladesh/bd.simple.org.chain.pem
+Decrypting bangladesh/bd.simple.org.pem.vault to bangladesh/bd.simple.org.pem
+Decrypting bangladesh/bd.simple.org.private_key.pem.vault to bangladesh/bd.simple.org.private_key.pem
+Decrypting bangladesh/terraform.tfvars.vault to bangladesh/terraform.tfvars
+Decrypting development/simple.org.chain.pem.vault to development/simple.org.chain.pem
+Decrypting development/simple.org.pem.vault to development/simple.org.pem
+Decrypting development/simple.org.private_key.pem.vault to development/simple.org.private_key.pem
+Decrypting development/terraform.tfvars.vault to development/terraform.tfvars
+Decryption complete!
+```
+
+Even if you've already decrypted these files, it's a good idea to do this again, as the contents of the encrypted files
+may have changed since you last decrypted them.
+
+### 3. Navigate to the AWS environment that you wish to work on.
+
+```bash
+$ cd bangladesh
+```
+
+### 4. If you haven't already, initialize Terraform. This command is safe to re-run several times.
+
+```bash
+$ terraform init
+```
+
+### 5. Encrypt any changed secrets
+
+If you modify a decrypted file during development, update the encrypted file and check it into the repository. For
+example, if you've modified `terraform.tfvars`,
+
+```bash
+$ cat terraform.tfvars | ansible-vault encrypt --vault-id ~/code/simple/ansible_password --output terraform.tfvars.vault
```

Comment:
Shall we stick to `~/.vault_password` since that seems to be the default in other places?

Reply:
Done.